addons/website_slides_survey/models/slide_slide.py

```diff
@@ -45,6 +46,29 @@ class Slide(models.Model):
         ('check_certification_preview', "CHECK(slide_type != 'certification' OR is_preview = False)", "A slide of type certification cannot be previewed."),
     ]
 
+    @api.constrains
+    def _check_certification_no_quiz(self):
+        for slide in self:
+            if slide.slide_type == 'certification' and slide.question_ids:
+                raise ValidationError(_("A certification slide can be linked to questions."))
+
+    # ---------------------------------------------------------
+    # ORM Overrides
+    # ---------------------------------------------------------
+
+    @api.model
+    def create(self, values):
```

Comment:
No multi? :(

Reply:
No, we start with no record to get one, so no api.multi.
And yes, we should use the 'create_multi' decorator, but as the one is slide is not using it (and not very efficient at all), we don't really care ^^"